usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -514,12 +514,16 @@ def get_or_create_object_class(row_object_class, row_direct_reimbursable, logger
     return obj_class
 
 
-def get_or_create_program_activity(program_activity_code):
+def get_or_create_program_activity(row, submission_attributes):
     # We do it this way rather than .get_or_create because we do not want to
     # duplicate existing pk's with null values
-    prg_activity = RefProgramActivity.objects.filter(ref_program_activity_id=program_activity_code).first()
-    if prg_activity is None and program_activity_code is not None:
-        prg_activity = RefProgramActivity.objects.create(ref_program_activity_id=program_activity_code)
```

Comment:
That's one of the legacy models. It was reflected out of an old postgres database, and it looks like no one went back and tuned it up. So, there's no good reason why it's not an Autofield. (Also that name is just terrible...)